internal/dag/httpproxy_processor.go

```diff
@@ -455,6 +455,8 @@ func (p *HTTPProxyProcessor) computeRoutes(
 			return nil
 		}
 
+		requestHashPolicies, lbPolicy := loadBalancerRequestHashPolicies(route.LoadBalancerPolicy, validCond)
```

Comment:
nit: I'd probably split this into two things, 1. determine the policy 2. gather the requestHashPolicies.

Reply:
i put these together since the policy/strategy will be different based on if the hash policies are valid or not (e.g. reset to `RoundRobin` if none of the hash policies are valid)